remote/bitbucketserver/types.go

```diff
@@ -84,89 +84,4 @@ type postHook struct {
 	} `json:"repository"`
 }
```

Comment:
this file should be removed all together and the structs in it could be moved to `internal/types.go`